server/src/main/resources/application-stage.yml

```diff
@@ -5,8 +5,8 @@ extender:
         type: S3
         s3.bucket: defold-extender-cache-stage
     remote-builder:
-        enabled: false
-        url: https://build-darwin-stage.defold.com
+        enabled: true
+        url: https://build-darwin.defold.com
```

Comment:
We really have to be extra careful now, since we cannot test things as we used to.